readme.md

```diff
@@ -162,13 +199,15 @@ Your Rails app has the following `application.html.erb`. Nothing shows up in you
 </html>
 ```
 
-> Your answer...
+The party hat is missing part of the hat in <% yield %>, the equal sign.
 
 ## Question 13
 
 Of the three options below, which is the most "correct" way of organizing the files that make up an Angular app, as used in class? Why is this option considered "better" than the other two?
 
-> Your answer...
+Option B is the correct way used in class. I'm still unsure why it's the convention since it seems it will get very cluttered very quickly when you have many directives and controllers, which I can imagine happens a lot in bigger apps.
+
+However, it's better than the other options because it keeps all controllers and views together so they can be accessed easier for maintenance, debugging, etc.
```

Comment:
I think the idea is that in Angular, features should be separated concerns instead of the traditional MVC.